web/src/lib/connectors/credentials.ts

```diff
@@ -311,8 +311,8 @@ export const credentialDisplayNames: Record<string, string> = {
   confluence_access_token: "Confluence Access Token",
 
   // Jira
-  jira_user_email: "Jira User Email",
-  jira_api_token: "Jira API Token",
+  jira_user_email: "Jira User Email (required for Jira Cloud)",
+  jira_api_token: "Jira API Token or Jira PAT",
```

Comment:
nit: "PAT" seems like a very power user'ish abbreviation to use ... would it be more intuitive to just spell out "Personal Access Token" instead?